Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py

```diff
@@ -459,21 +294,23 @@ def get_company_findings_command(client, args, first_seen=None, fetch_incidents=
             res = client.get_companies_guid()
             if res.status_code == 200:
                 res_json = res.json()
-                guid = res_json.get('my_company')['guid']
+                guid = res_json.get('my_company', {}).get('guid')
             else:
                 raise Exception('Unable to fetch GUID')
         severity = args.get('findings_min_severity', None)
         if severity:
             severity = severity.lower()
         grade = args.get('findings_grade', None)
         if type(grade) is list:
-            grade = grade.join(',')
+            grade = ','.join(grade)
```

Comment:
This will definitely not work if grade is a list. Please explain what you are trying to achieve and I will try to help

Reply:
I was trying to convert the list into comma separated string. For example converting list ['a','b'] into string 'a,b'.

Reply:
OK. So out arguments are always of type string so this shouldn't be a problem